web/admin/application/configs/klear/HuntGroupsList.yaml

```diff
@@ -98,33 +98,41 @@ production:
     <<: *huntGroupsRelUsers_screensLink
     huntGroupsRelUsersList_screen:
       <<: *huntGroupsRelUsersList_screenLink
-      title: _("List of %s %2s", ngettext('User', 'Users', 0), "[format| (%parent%)]")
-      class: ui-silk-user
+      title: _("List of %s %2s", ngettext('Member', 'Members', 0), "[format| (%parent%)]")
+      class: ui-silk-group-go
       filterField: HuntGroup
       parentOptionCustomizer:
         - IvozProvider_Klear_Options_OptionsCustomizerFilterForHuntGroup
         - recordCount
+
     huntGroupsRelUsersEdit_screen:
       <<: *huntGroupsRelUsersEdit_screenLink
-      title: _("Edit %s", ngettext('User', 'User', 1))
+      title: _("Edit %s", ngettext('Member', 'Members', 1))
       filterField: HuntGroup
+
     huntGroupsRelUsersNew_screen:
       <<: *huntGroupsRelUsersNew_screenLink
-      title: _("Add %s %2s", ngettext("User", "users", 1), "[format| (%parent%)]")
+      title: _("Add %s %2s", ngettext('Member', 'Members', 1), "[format| (%parent%)]")
       filterField: HuntGroup
 
     # HuntGroupRelUsers filter by "ringAll"
     huntGroupsRelUsersFilterRingAllList_screen:
       <<: *huntGroupsRelUsersList_screenLink
-      title: _("List of %s %2s", ngettext('User', 'Users', 0), "[format| (%parent%)]")
-      class: ui-silk-user
+      title: _("List of %s %2s", ngettext('Member', 'Members', 0), "[format| (%parent%)]")
```

Comment:
Should we update HuntGroupsRelUsers.name on PublicEntities table?

Reply:
Renaming the entity from HuntGroupRelUsers to HuntGroupMembers will be handled in https://github.com/irontec/ivozprovider/issues/1303

I have added a subtask for PublicEntities to the list